src/api/BaseClient.js

```diff
@@ -0,0 +1,129 @@
+import queryString from "query-string";
```

Comment:
Should this be a hook ?

Reply:
`query-string` is a third party package: https://www.npmjs.com/package/query-string